bait-files/src/main/webapp/submission.js

```diff
@@ -0,0 +1,209 @@
+function loadAmericanAddress() {
+
+    let addressInputDiv = document.getElementById("address-input-div");
+
+    //Address
```

Comment:
Would it be possible to have address split into 2 inputs? Street Line 1 (100 test drive) and Street Line 2 (Apartment 12, etc where the second input could be null?). This would be good for using any APIs that rely on address input. If you do go this route it would be important to use a placeholder for these inputs to show what each one should have.

Reply:
Can do

Reply:
Changed!